package.json

```diff
@@ -56,6 +58,9 @@
     "gulp-rename": "^1.2.0",
     "gulp-uglify": "^1.0.2",
     "gulp-watch": "^3.0.0",
+    "qunitjs": "^2.2.1",
```

Comment:
It's a peer dependency for gulp-test, when doing a fresh install npm throw a warning it's missing in our package.json.